web/src/helpers/themes.js

```diff
@@ -34,8 +34,8 @@ export const themes = {
   },
   bright: {
     co2Scale: {
-      steps: [0, 150, 600, 750, 800],
-      colors: ['#2AA364', '#F5EB4D', '#9E4229', '#381D02', '#381D02'],
+      steps: [0, 300, 600, 900, 1200, 1500],
+      colors: ['#2AA364', '#F5EB4D', '#C6983A', '#885022', '#381D02', '#381D02'],
```

Comment:
To follow up on my comment, here's an example of what I was trying to say 😄 

How about something like this instead? Also seems like we shouldn't use the same color twice at the end, it only made sense before because of the 750/800 step being so close (don't know why it was like that actually).

```suggestion
      colors: ['#2AA364', '#F5EB4D', '#B46C32', '#6B3015', '#2A1602', '#000000']
```

<img width="1189" alt="Screenshot 2022-11-22 at 22 59 32" src="https://user-images.githubusercontent.com/3296643/203430192-def38503-5529-4bb8-9844-94e630bc2132.png">